### PR TITLE
Backend added tags documentation

### PR DESCRIPTION
**Pull request by dussault-antoine**
### What does this PR do?

Document backend added span tags as unusable to exclude traces at the Agent Level

### Motivation

https://datadoghq.atlassian.net/browse/APMS-7745

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by yoannrobin — APPROVED** [review]

File: content/en/tracing/guide/ignoring_apm_resources.md [hunk 1]
```diff
@@ -54,12 +54,26 @@ apm_config:
     reject: ["http.url:http://localhost:5050/healthcheck"]
 {{< /code-block >}}
 
-<div class="alert alert-warning"><strong>Note</strong>: Filtering traces this way removes these requests from <a href="/tracing/guide/metrics_namespace/">trace metrics</a>. For information on how to reduce ingestion without affecting the trace metrics, see <a href="/tracing/trace_ingestion/ingestion_controls">ingestion controls</a>.</div>
+**Note**: Filtering traces this way removes these requests from [trace metrics][3]. For information on how to reduce ingestion without affecting the trace metrics, see [ingestion controls][4]
```
Comment (alai97) [inline content/en/tracing/guide/ignoring_apm_resources.md:57]:
```suggestion
Filtering traces this way removes these requests from [trace metrics][3]. For more information on how to reduce ingestion without affecting the trace metrics, see [Ingestion Controls][4].
```

File: content/en/tracing/guide/ignoring_apm_resources.md [hunk 2]
```diff
@@ -54,12 +54,26 @@ apm_config:
...
+
+The following span tags are created and added to spans after ingestion on the Datadog backend side. Hence, these tags cannot be used to drop traces at the Datadog Agent level.
```
Comment (alai97) [inline content/en/tracing/guide/ignoring_apm_resources.md:59]:
```suggestion
On the backend, Datadog creates and adds the following span tags to spans after ingestion. These tags cannot be used to drop traces at the Datadog Agent level.
```

File: content/en/tracing/guide/ignoring_apm_resources.md [hunk 3]
```diff
@@ -54,12 +54,26 @@ apm_config:
...
+|-----------------------------------------|--------------------------------------------------|
+| `http.path_group`                       | The full url path from the `http.url` tag        |
+| `http.url_details.host`                 | The host name portion of the `http.url` tag      |
+| `http.url_details.path`                 | The full request target as passed in a HTTP request line or equivalent |
+| `http.url_details.scheme`               | The request scheme from the `http.url` tag       |
+| `http.url_details.queryString`          | The query string portion from the `http.url` tag |
+| `http.url_details.port`                 | The HTTP port from the `http.url` tag            |
+| `http.useragent_details.os.family`      | The OS family reported by the User-Agent         |
+| `http.useragent_details.browser.family` | The Browser Family reported by the User-Agent    |
+| `http.useragent_details.device.family`  | The Device family reported by the User-Agent     |
```
Comment (alai97) [inline content/en/tracing/guide/ignoring_apm_resources.md:72]:
```suggestion
| `http.path_group`                       | The full URL path from the `http.url` tag.        |
| `http.url_details.host`                 | The host name portion of the `http.url` tag.      |
| `http.url_details.path`                 | The full request target as passed in an HTTP request line or equivalent. |
| `http.url_details.scheme`               | The request scheme from the `http.url` tag.       |
| `http.url_details.queryString`          | The query string portion from the `http.url` tag. |
| `http.url_details.port`                 | The HTTP port from the `http.url` tag.            |
| `http.useragent_details.os.family`      | The OS family reported by the User-Agent.         |
| `http.useragent_details.browser.family` | The browser family reported by the User-Agent.    |
| `http.useragent_details.device.family`  | The device family reported by the User-Agent.     |
```

**Review by alai97 — CHANGES_REQUESTED** [review]
Some copy suggestions!

**Review by alai97 — APPROVED** [review]

**Pull request merged**